source/whats-new.txt

```diff
@@ -59,6 +60,94 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.9:
+
+What's New in 6.9
+-----------------
+
+- Support for {+mdb-server+} 3.6 is deprecated and will be removed in a future driver release.
+
+- The driver natively supports explicit resource management for cursors and the
+  ``MongoClient``, ``ClientSession``, and ``ChangeStream`` objects. When using a compatible Node.js
+  version, you can implement explicit resource management by using the ``cursor.stream()``
+  method and ``GridFSDownloadStream`` class.
+
+  To learn how to use explicit resource management, see the
+  :github:`v6.9.0 Release Notes <mongodb/node-mongodb-native/releases/tag/v6.9.0>` on
+  GitHub. For more information about explicit resource management, see the
+  :github:`ECMAScript Explicit Resource Management Proposal </tc39/proposal-explicit-resource-management>`
+  on GitHub.
+
+- If you're using Node.js 18.13 or later, you can pass the ``autoSelectFamily`` and
+  ``autoSelectFamilyTimeout`` options to the ``MongoClient`` constructor. When the
+  ``autoSelectFamily`` option is ``true``, the driver automatically selects between
+  an IPv4 or IPv6 connection. The ``autoSelectFamilyTimeout`` option specifies the
+  timeout, in milliseconds, for the driver to select the connection family.
+
+  The following code example shows how to use these options:
+
+  .. code-block:: javascript
+     :copyable: true
+
+     const client = new MongoClient(url,
+       { autoSelectFamily: true, autoSelectFamilyTimeout: 100 });
+
+- The ``MongoClient`` constructor accepts the ``allowPartialTrustChain`` option. This option
+  controls the ``X509_V_FLAG_PARTIAL_CHAIN`` OpenSSL flag.
+
+- The ``ConnectionCheckedOutEvent`` and ``ConnectionCheckFailedEvent`` now include a ``durationMS``
+  property. This property indicates the time elapsed between the start and end of a
+  connection checkout operation.
```

Comment:
S: I believe this change require updates to this section: https://deploy-preview-902--docs-node.netlify.app/fundamentals/monitoring/connection-monitoring/#connectioncheckedout

Reply:
Thanks for pointing that out. It's a strange situation when the original ticket explicitly says docs changes [Not Needed](https://jira.mongodb.org/browse/NODE-5459), because it adds a lot of time to version update work when there should have just been a Docs ticket in the first place